packages/adventure.rb

```diff
@@ -0,0 +1,14 @@
+require 'package'
```

Comment:
We can remove this line since it will load in `libbsdgames2.rb`

Reply:
Good point.

Reply:
621edfc